run_unittests.py

```diff
@@ -4386,6 +4386,14 @@ def test_version_checked_before_parsing_options(self):
         match = 'Meson version is.*but project requires >=2000'
         self.assertMesonRaises("", match, meson_version='>=2000', options=options)
 
+    def test_assert_default_message(self):
+        self.assertMesonRaises('''
+            k1 = 'a'
+            assert({
+              k1: 1,
+              'b': 2
+            }['a'] == 2)''',
+            "Assert failed: {k1 : 1, 'b' : 2}\['a'\] == 2")
```

Comment:
__[Flake8]__ continuation line under-indented for visual indent ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6285#issue-4893477))

<details>

| Rule |
| - |
| E128 |

</details>
<sub>posted by Sider</sub>